modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-baremetal_{context}"]
+= Configuring registry storage for bare metal
+
+During installation, you must configure your registry to use storage.
+
+.Prerequisites
+
+* A cluster on bare metal.
+* PV with `ReadWriteMany` access mode.
+* Must have "100Gi" capacity.
+
+.Procedure
+
+. To configure your registry to use storage, change the `spec.storage.pvc` in
+the `configs.imageregistry/cluster` resource.
+
+. Provide a suitable persistent volume, such as Azure File or NFS.
+
+. Verify you do not have a registry pod:
++
+----
+$ oc get configs.imageregistry.operator.openshift.io/cluster -o yaml
+----
+
+. Check the registry configuration:
++
+----
+$ oc edit configs.imageregistry.operator.openshift.io
+
+storage:
+----
+
+. Check the `clusteroperator` status:
++
+----
+$ oc get clusteroperator image-registry
+----
++
+There will be warning similar to:
++
+----
+- lastTransitionTime: 2019-03-26T12:45:46Z
```

Comment:
This makes it look like your storage is still not configured. Is that right? If you wait and run the command again, should you see a different message?